test/linter/test-versions.js

```diff
@@ -66,15 +66,15 @@ const blockMany = [
 ];
 
 /** @type {object.<string, string[]>} */
-const blockList = {
-  api: blockMany,
-  css: blockMany,
+const realValuesRequired = {
```

Comment:
Ditto, is this bascially `browsersRequiringNumericalVersionsBySection` (assuming that "api", "css" etc. are sections, not sure if this is the right term)?